src/components/Header/Header.tsx

```diff
@@ -0,0 +1,65 @@
+import classNames from 'classnames';
+import React, { useEffect, useRef, useState } from 'react';
+
+type Props = {
+  addTodo: (title: string) => void;
+  hasActive: boolean;
+  isInputActive: boolean;
+  hasTodos: boolean;
+  toggleAll: () => void;
+};
+
+export const Header: React.FC<Props> = ({
+  addTodo,
+  hasActive,
+  isInputActive,
+  hasTodos,
+  toggleAll,
+}) => {
+  const [title, setTitle] = useState('');
+  const todoInput = useRef<HTMLInputElement | null>(null);
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    addTodo(title);
+    setTitle('');
+  };
+
+  useEffect(() => {
+    if (todoInput.current) {
```

Comment:
mb here should be condition for `&& isInputActive`?
for me it seems like you trying to trigger some state and after ping your input, but would be better to pass `ref` to this component and create some function on parent level

Reply:
Not resolved, try to change your logic like proposed in the comment, if you have any questions - feel free to ask them in `fe_chat`